src/V3EmitMk.cpp

```diff
@@ -24,18 +24,514 @@
 
 VL_DEFINE_DEBUG_FUNCTIONS;
 
+class OutputGroup final {
```

Comment:
```suggestion
// Add comments about what does
class EmitGroup final {
```

(Prefer leading word in classes matches filename)